### ISSUE TITLE
Add an inline "Go to original post to see all replies" message when viewing remote toots

### ISSUE THREAD

**Issue by audiodude** [mastodon/mastodon#26863]
### Pitch

We propose adding an inline hint when viewing a toot that allows the user to be aware of the problem and find all replies easily:

![image](https://github.com/mastodon/mastodon/assets/57832/f70a86c2-518d-462d-a4a9-69ff0934ab1e)

### Motivation

Full problem description with bug links and proposed solution here:

https://docs.google.com/document/d/1YpdrZVTrk-yt0GpnSxuDybYvFQOA0B-IfkELBYNwSOg/edit

TLDR:

When a user views a toot today, they are likely to miss some if not most of the replies to the toot. This is especially pronounced for users on smaller instances. The reason for this is that the user’s own server only holds replies to the parent toot from accounts that are followed by someone on that user’s server.

**Comment by shleeable** [comment]
I'd rather have a "sync toots" button, to pull in replies.

**Comment by gabeklavans** [comment]
I think this is a pretty big issue, and a decent solution for now. It's pretty disheartening to know that I'm missing ~30% of all replies when I use mastodon on my relatively mid-large sized instance. I'm sure a "sync" button has a slew of technical challenges itself, so I'd like this change as a stop-gap.
